openapi.yaml

```diff
@@ -14336,6 +15043,224 @@ components:
+        rules:
+          type: object
+          description: |
+            The inbound and outbound access rules to apply to the Firewall.
+
+              * A minimum of one open inbound rule is required. Any inbound
+                traffic that is not permitted by your rules will be blocked.
+              * Outbound rules are optional. When no outbound rules are specified,
+                all outbound traffic is allowed. If one or more outbound rules are
+                specified, all outbound traffic that is not permitted by your rules
+                will be blocked.
+
+            A Firewall may have up to 25 rules across its inbound and outbound rulesets.
+          properties:
+            inbound:
+              type: array
+              items:
+                $ref: '#/components/schemas/FirewallRuleConfig'
+            outbound:
+              type: array
+              items:
+                $ref: '#/components/schemas/FirewallRuleConfig'
+          required:
+            - inbound
+        tags:
+          x-linode-filterable: true
+          description: >
+            An array of tags applied to this object. Tags are for organizational
+            purposes only.
+          type: array
+          items:
+            type: string
+          example:
+          - example tag
+          - another example
+    FirewallRuleConfig:
+      type: object
+      description: >
+        One of a Firewall's inbound or outbound access rules. The `ports` property
+        can be used to allow traffic on a comma-separated list of different ports.
+      required:
+      - protocol
+      properties:
+        protocol:
+          type: string
+          enum:
+          - TCP
+          - UDP
+          - ICMP
+          description: >
+            The type of network traffic to allow.
+          example: TCP
+        ports:
+          type: string
+          description: |
+            A string representing the port or ports on which traffic will be allowed:
+
+            - The string may be a single port, a range of ports, or a comma-separated list
+            of single ports and port ranges. A space is permitted following each comma.
+            - A range of ports is inclusive of the start and end values for the range. The
+            end value of the range must be greater than the start value.
+            - Ports must be within 1 and 65535.
+            - Ports may not be specified if a rule's protocol is `ICMP`. At least one port
+            must be specified if a rule's protocol is `TCP` or `UDP`.
+            - The ports string can have up to 15 *pieces*, where a single port is treated
+            as one piece, and a port range is treated as two pieces. For example,
+            the string "22-24, 80, 443" has four pieces.
+          example: '22-24, 80, 443'
+        addresses:
+          type: object
+          description: >
+            Whitelisted IPv4 or IPv6 addresses. A Rule can have up to 255 addresses
```

Comment:
```suggestion
            Allowed IPv4 or IPv6 addresses. A Rule can have up to 255 addresses
```

Reply:
Updated with your suggestion. Thanks so much for reviewing @alsaihn !!!!